.github/workflows/ci.yml

```diff
@@ -12,7 +12,6 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-11
           - macos-12
           - macos-13
           - macos-13-xlarge
```

Comment:
I'm a bit afraid of that.
kiwix-build generate dependencies build for macos-13 but publish them using the name `macos` only (without version).

Should we removed macos-12 ?
If not, it would be better to make kiwix-build also generate macos-12 dependencies and add the mac-os version to the archive name.

Reply:
> I'm a bit afraid of that. kiwix-build generate dependencies build for macos-13 but publish them using the name `macos` only (without version).
> 
> Should we removed macos-12 ? If not, it would be better to make kiwix-build also generate macos-12 dependencies and add the mac-os version to the archive name.
> 
> Edit: Well, ios CI is actually failing because `meson_cross_file.txt` points to compiler not existing in macos-12. We have to do something here:
> 
>     * Either build IOS on macos-13 (and drop macos-12)
> 
>     * Or build deps on macos-12 too.

@mgautierfr We could and [should remove macos-12 here and compile only on `macos-13`](https://github.com/kiwix/apple/pull/533), BUT then we have to put the right compilation parameter to secure macOS12 will be supported at runtime.

Reply:
In https://github.com/kiwix/kiwix-build/pull/650 we set `-mmacosx-version-min` to `12.0` so we should be ok.

Reply:
We are good there... but here?

Reply:
> Well, ios CI is actually failing because `meson_cross_file.txt` points to compiler not existing in macos-12. We have to do something here:
> 
>     * Either build IOS on macos-13 (and drop macos-12)
> 
>     * Or build deps on macos-12 too.

This is https://github.com/openzim/libzim/issues/834. I guess moving to macos13 resolves the problem?! :)